src/python/strelka/scanners/scan_ocr.py

```diff
@@ -1,3 +1,4 @@
+import fitz
```

Comment:
I believe CI is failing because it doesn't think it can find this module. The `fitz` module is included in the `PyMuPDF` module https://pymupdf.readthedocs.io/en/latest/module.html. I can add a comment to skip the style check if that's desired 👍